portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -19,9 +19,11 @@
 
   private final long id;
   private final String commentString;
+  private final int score;
 
-  public Comment(long id, String commentString) {
+  public Comment(long id, String commentString, int score) {
     this.id = id;
     this.commentString = commentString;
```

Comment:
No need to add String to the name of variables since the type already implies it.